website/source/docs/platform/k8s/injector/index.html.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: "docs"
+page_title: "Agent Sidecar Injector Overview"
+sidebar_current: "docs-platform-k8s-injector"
+sidebar_title: "Agent Injector"
+description: |-
+  The Vault Agent Sidecar Injector is a Kubernetes admission webhook that adds Vault Agent containers to pods for consuming Vault secrets.
+---
+
+# Agent Sidecar Injector
+
+The Vault Agent Injector alters pod specifications to include Vault Agent
+containers that render Vault secrets to a shared memory volume using 
+[Vault Agent templates](/docs/agent/template/index.html).
+By rendering secrets to a shared volume, containers within the pod can consume 
+Vault secrets without being Vault aware.
+
+The injector is a [Kubernetes Mutation Webhook Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+The controller intercepts pod events and applies mutations to the pod if annotations exist within
+the request.  This functionality is provided by the [vault-k8s](https://github.com/hashicorp/vault-k8s)
+project and can be automatically installed and configured using the
```

Comment:
> provided by the vault-k8s project

Calling `vault-k8s` a _project_ seems odd here, is there a technical name for something that runs webhook controllers?

Reply:
Yes, `vault-k8s` ;)